app/scripts/controllers/overview.js

```diff
@@ -692,10 +697,82 @@ function OverviewController($scope,
     return true;
   };
 
+  // part of lodash since 4.0.0
```

Comment:
We're using lodash 4, so you should be able to use `_.keyBy`

Reply:
done